src/predict_3dpose.py

```diff
@@ -75,12 +75,19 @@
   'batch_normalization' if FLAGS.batch_norm else 'no_batch_normalization',
   'use_stacked_hourglass' if FLAGS.use_sh else 'not_stacked_hourglass',
   'predict_14' if FLAGS.predict_14 else 'predict_17')
-
+train_dir='train'
 print( train_dir )
 summaries_dir = os.path.join( train_dir, "log" ) # Directory for TB summaries
 
 # To avoid race conditions: https://github.com/tensorflow/tensorflow/issues/7448
-os.system('mkdir -p {}'.format(summaries_dir))
+def mkdir_p(path):
```

Comment:
Thank you. Have you tested this on a windows cluster?
The race condition I reported (https://github.com/tensorflow/tensorflow/issues/7448) seems to occur only when multiple jobs are writing to the same sub-directory. I'm not sure this code is solving that problem more generally.